analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Monitor.Conditions.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Net;
```

Comment:
This doesn't seem intentional. Missing selfreview?
```suggestion
```

Reply:
Just missed it.